backend/src/models/reports/Tab_4_rpt_CA_Capital_GDX.js

```diff
@@ -1,16 +1,193 @@
-//const dbConnection = require("@database/databaseConnection");
-//const { knex } = dbConnection();
+const dbConnection = require("@database/databaseConnection");
+const { knex } = dbConnection();
+const { groupByProperty } = require("../../controllers/reports/helpers");
+const _ = require("lodash");
 
-const getReport = (portfolio) => {
-  const query = "removeme"; //knex().raw();
-  return query;
+/**
+ * Retrieves the Capital GDX data, filtered by fiscal
+ *
+ * @param   {number}  fiscal - The fiscal year to grab data for
+ * @returns {Promise}        - A promise that resolves to the query result
+ */
+
+const reportQueries = {
+  fiscalYear: (fiscal) =>
+    knex("fiscal_year").select("fiscal_year").where("fiscal_year.id", fiscal).first(),
+
+  // The columns on which to calculate totals.
+  columns: {
```

Comment:
you're getting a lot of mileage outta this - maybe columns should be in the boilerplate. I'll think about that